### ISSUE TITLE
Breakpoints moving to end of the file

### ISSUE THREAD

**Issue by tonysneed** [Microsoft/vscode-chrome-debug#468]
- VS Code Version: 1.14.2
- Angular CLI Version: 1.1.1
- Launch.json:

```json
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "chrome",
            "request": "launch",
            "name": "ng serve",
            "url": "http://localhost:4200/#",
            "webRoot": "${workspaceRoot}"
        }
}
```

Steps to reproduce:

1. Set breakpoints in ts files.
2. Run `npm start` (`ng serve`).
3. Launch debugger in VS Code (F5).

At this point all my breakpoints move to the end of the file.

<img width="905" alt="breakpoint-moved" src="https://user-images.githubusercontent.com/2836367/28716185-5815fd90-7361-11e7-90ba-5988017a0702.png">

4. Execute code path (button click). Code breaks at unexpected location.

<img width="1045" alt="code-break" src="https://user-images.githubusercontent.com/2836367/28716230-a21fecca-7361-11e7-8e5d-1a5f2dd01a62.png">

**Comment by tonysneed** [comment]
I don't see this happening anymore after upgrading my CLI version to [1.3-rc3](https://github.com/angular/angular-cli/releases/tag/v1.3.0-rc.3).

**Comment by auchenberg** [comment]
@tonysneed Interesting. Looks like a lot of sourcemap/webpack changes has landed in 1.3-rcX. https://github.com/angular/angular-cli/releases
